### DISCUSSION TITLE
specific views of events in widget

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3072] [Q&A]
Hello.

Can I create views that contain specific events in pretix ?
Let's say I have created 4 events (A,B,C,D) and I want to display only the A,B,C events in a widget calendar.
Is this possible ?

**Comment by raphaelm** [comment]
Yes, it's possible, see the last example at
https://docs.pretix.eu/en/latest/user/events/widget.html#multi-event-selection
